gulpfile.coffee

```diff
@@ -103,6 +107,7 @@ gulp.task 'dist', ['clean:dist'], ->
     '*.html'
     'package.json'
     'example.md'
+    'sass/katex/fonts/*'
```

Comment:
Just a little more! This line is no longer using.
I'll merge after you remove this line. I'm grateful for you work! 😆